pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -2326,6 +2327,74 @@ def test_datetimeindex(self):
         for idx in [idx1, idx2, idx3]:
             tm.assert_index_equal(idx, expected)
 
+    def test_bday_ignores_timedeltas(self):
+        idx = pd.date_range("2010/02/01", "2010/02/15", freq="6H")
+        t1 = idx + BDay(offset=pd.Timedelta(3, unit="H"))
+
+        expected = DatetimeIndex(
```

Comment:
i don't feel strongly about this but is there not a more concise way to generate this

Reply:
Reducing the date range seems to be the way to go here. Maybe extend the freq too?

Reply:
Done